pkg/algorithms/cmr/cmr.py

```diff
@@ -0,0 +1,123 @@
+"""EDivisive Algorithm from hunter"""
+
+# pylint: disable = line-too-long
+from typing import List
+import pandas as pd
+import numpy
+
+from fmatch.logrus import SingletonLogger
+from hunter.series import  ChangePoint, ComparativeStats
+from pkg.algorithms.algorithm import Algorithm
+
+
+class CMR(Algorithm):
+    """Implementation of the CMR algorithm
+    Will Combine metrics into 2 lines and compare with a tolerancy to set pass fail
+
+    Args:
+        Algorithm (Algorithm): Inherits
+    """
+
+
+    def _analyze(self):
+        """Analyze the dataframe with meaning any previous data and generate percent change with a current uuid
+
+        Returns:
+            series: data series that contains attributes and full dataframe
+            change_points_by_metric: list of ChangePoints
+        """
+        logger_instance = SingletonLogger.getLogger("Orion")
+        logger_instance.info("Starting analysis using Isolation Forest")
```

Comment:
just curious, what does CMR stand for, Is it same as isolation forest?

Reply:
Honestly not 100% sure, put "Comparing Mean Results" instead